src/seth/libexec/seth/seth-source

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+### seth-source -- fetch the contract source from etherscan
+### Usage: seth source <address> [<options>]
+###
+### Requires an etherscan api key
+###
+set -e
+[[ $ETHERSCAN_API_KEY ]] || {
+  cat >&2 <<.
```

Comment:
Is it possible shellcheck doesn't like `<<.`? I've seen it choke on it elsewhere, and using any other terminator (like the venerable `EOF`) made it happy.